src/color-chart/color-chart.js

```diff
@@ -128,7 +129,22 @@ const Self = class ColorChart extends ColorElement {
 			ret.colors[index] = color = color.to(this.space);
 			ret.swatches.set(color, swatch);
 
-			let x = Number(name.match(/-?\d*\.?\d+$/)?.[0] ?? index);
+			let x = name.match(/-?\d*\.?\d+$/)?.[0];
+			if (x !== undefined) {
+				// Transform `Label / X-coord` to `Label`
+				// (there should be at least one space before and after the slash so the number is treated as an X-coord)
+				let regexp = new RegExp(`\\s+\\/\\s+${x}$`);
+				name = name.replace(regexp, "").trim();
```

Comment:
We now have the number, so we can match more efficiently, no need to check for *any* number. We can just slice by the length of the number, trim, and then check if that ends with a slash.

Reply:
Makes perfect sense. I fixed it. Thanks!